json_schemas/core.schema.json

```diff
@@ -0,0 +1,185 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://w3c.github.io/did-spec-registries/schemas/core.schema.json",
```

Comment:
Ideally this would be a nice self referencing URL, @iherman can you take another stab at this?